python/lsst/analysis/tools/actions/vector/selectors.py

```diff
@@ -616,10 +616,8 @@ class ParentObjectSelector(FlagSelector):
     def setDefaults(self):
         # This selects all of the parents
         self.selectWhenFalse = [
-            "detect_isDeblendedModelSource",
             "sky_object",
         ]
-        self.selectWhenTrue = ["detect_isPatchInner"]
```

Comment:
Should there be any changes to coaddObjectSelector?

Reply:
The `CoaddPlotFlagSelector`? No, I don't think so. I know that you kind of came in the middle of the review without seeing the upstream changes but those columns still exist in the `object` table, just not the new `object_parent` table that is only used for the deblender metrics.